src/ucp/api/ucp_def.h

```diff
@@ -250,6 +273,33 @@ typedef void (*ucp_request_cleanup_callback_t)(void *request);
 typedef void (*ucp_send_callback_t)(void *request, ucs_status_t status);
 
 
+ /**
+ * @ingroup UCP_COMM
+ * @brief Callback to process peer failure.
+ *
+ * This callback routine is invoked when transport level error detected.
+ *
+ * @param [in]  arg      User argument to be passed to the callback.
+ * @param [in]  ep       Endpoint to handle transport level error,
+ *                       @a ep becomes unusable. All following operation on @a ep
+ *                       will return @a status error.
```

Comment:
Please update with wording similar to what we used in uct_def.h for #1437:
Endpoint to handle transport level error.  Upon return from the callback, this @a ep is no longer usable and all subsequent operations on this @a ep will fail with the error code passed in @a status